### DISCUSSION TITLE
restrictions on filtered results

### DISCUSSION THREAD

**Discussion by agatom160** [microsoft-search/pnp-modern-search#3195] [Q&A]
We have a question about the featured extension https://microsoft-search.github.io/pnp-modern-search/. The add-on was installed on our environment and after the last tests, a question arose about the possibility of limiting the filtered results to the folder level only.
Example:
We search for files in the context of xyz company and selected review target (RefinableString03 and RefinableString00, respectively).

![screen](https://github.com/microsoft-search/pnp-modern-search/assets/142575097/4d998723-1f73-4a2d-aa95-673a425f6baa)

The system now displays a specific file or files in the search results.

![screen2](https://github.com/microsoft-search/pnp-modern-search/assets/142575097/aff1baa3-d6f1-4bb9-a317-aca279446438)

The effect we would like to achieve would be to display in the results list folders containing files that meet the search and filter conditions:

example:
![screen3](https://github.com/microsoft-search/pnp-modern-search/assets/142575097/aa3aed26-6371-46f3-920b-28e3cf5d2f20)

Recommendation on how to achieve the above result?

**Comment by wobba** [comment]
In the query template field of the web part add this KQL:

`{searchTerms} HtmlFileType<>"OneNote.Notebook" AND ContentTypeId:0x0120* `

This should return folders only.

**Reply by agatom160** [reply]
Thank you Mikael! 
Your solution works very well.
However, the issue pertains to the fact that the refiner doesn't recognize folders; it only detects files with various extensions.
We are wondering if this is a property of the refiner or maybe a filtering problem that doesn't allow the use of refiners to filter folders. 
do you know a solution?

**Reply by wobba** [reply]
A bit hard to know what properties are used in your RefinableString properties. And if the values from them are on the folders itself I don't see why it shouldn't work.
